PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Data/Data/HospitalDbContext.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using PrivateHospitals.Core.Models;
+
+namespace PrivateHospitals.Data.Data;
+
+public class HospitalDbContext: IdentityDbContext<AppUser>
+{
+    public HospitalDbContext(DbContextOptions<HospitalDbContext> options): base(options)
+    {
+    }
+    
+    public HospitalDbContext()
+    {
+    }
+    
+    public DbSet<Patient> Patients { get; set; }
+    public DbSet<Doctor> Doctors { get; set; }
```

Comment:
Both doctors and patiens are Users, but you don't specify it anywhere. Your users will have its roles and this way you can identify it.
Read about EF TPC, TPH approaches and choose way to represent your classes in DB. Because current approach will not work - Users should be Doctors and Patients

Reply:
Fixed